### PR TITLE
Auth Endpoint create user

### PR DESCRIPTION
**Pull request by zhaojzn**

### CODE REVIEW

File: src/GoogleSheets/packages/auth/create/main.go [hunk 1]
```diff
@@ -22,11 +25,30 @@ func HandleRequest(ctx context.Context, event events.APIGatewayProxyRequest) (ev
...
+	employeeId, err := GetEmployeeId.HandleRequest(signReq.Email)
+	if err != nil {
+		return events.APIGatewayProxyResponse{
+			StatusCode: 500,
+			Body:       fmt.Sprintf("Error getting employeeId: %v", err.Error()),
+		}, nil
+	}
+
+	fmt.Println("Employee ID:", employeeId)
+	clientId := os.Getenv("CLIENT_ID")
```
Comment (jlui17) [inline src/GoogleSheets/packages/auth/create/main.go:47]: Can we rename the env variable to smth like "COGNITO_CLIENT_ID"? Also, let's do a check to see if it's an empty string. If it's empty, let's return a 500 response saying smth like "No cognito client id found"

File: src/GoogleSheets/packages/new_auth/main.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+			StatusCode: 500,
+			Body:       fmt.Sprintf("Error signing up: %v", err.Error()),
+		}, nil
+	}
+
+	response := NewAuthConstants.SignUpResponse{
+		NeedsConfirmation: !(*signUpOutput.UserConfirmed),
+	}
+
+	reponseBody, err := json.Marshal(response)
```
Comment (jlui17) [inline src/GoogleSheets/packages/new_auth/main.go]: Let's do an error check here for safe measure

File: src/GoogleSheets/packages/auth/create/main.go [hunk 3]
```diff
@@ -44,7 +44,14 @@ func HandleRequest(ctx context.Context, event events.APIGatewayProxyRequest) (ev
 	}
 
 	fmt.Println("Employee ID:", employeeId)
-	clientId := os.Getenv("CLIENT_ID")
+	clientId := os.Getenv("COGNITO_CLIENT_ID")
```
Comment (jlui17) [inline src/GoogleSheets/packages/auth/create/main.go:47]: I think we need to change the var name in the cdk as well

**Review by jlui17 — APPROVED** [review]
LGTM

**Comment by kel-z** [comment]
pogger 👍

**Pull request merged**